data/dummydata.v2.js

```diff
@@ -1,1724 +1,48 @@
+const donuts = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/donuts1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/donuts2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/donuts3.jpeg"];
+const iceCream = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/icecream1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/icecream2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/icecream3.jpeg"];
+const indian = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/indian1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/indian2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/indian3.jpeg"];
+const pasta = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta1.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta2.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pasta3.jpg"];
+const pizza = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/pizza1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pizza2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/pizza3.jpeg"];
+const sandwich = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/sandwich1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/sandwich2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/sandwich3.jpeg"];
+const sushi = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/sushi1.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/sushi2.jpg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/sushi3.jpg"];
+const taco = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/taco1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/taco2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/taco3.jpeg"];
+const thai = ["https://s3-us-west-1.amazonaws.com/imagesfornamson/thai1.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/thai2.jpeg", "https://s3-us-west-1.amazonaws.com/imagesfornamson/thai3.jpeg"];
 
-var allItems = boba.concat(burger).concat(chinese).concat(coffee).concat(dimSum).concat(donuts).concat(iceCream).concat(indian).concat(pasta).concat(pizza).concat(sandwich).concat(sushi).concat(taco).concat(thai);
+const allItems = boba.concat(burger).concat(chinese).concat(coffee).concat(dimSum).concat(donuts).concat(iceCream).concat(indian).concat(pasta).concat(pizza).concat(sandwich).concat(sushi).concat(taco).concat(thai);
 
 
 
 
 
-for (var i = 0; i < dummy.length; i++) {
-  dummy[i].about.description = faker.lorem.paragraphs();
-  dummy[i].about.phone = faker.phone.phoneNumberFormat();
-  dummy[i].about.hours = randomElement(hours);
-  dummy[i].about.style = randomElement(styles);
-  dummy[i].about.price = randomElement(price);
-  dummy[i].banner = randomElement(int);
+for (let i = 0; i < dummy.length; i++) {
```

Comment:
I think beth suggested to do this in a .forEach()

Reply:
updated

Reply:
👍